app/controllers/api/account.php

```diff
@@ -56,8 +56,9 @@
     ->label('sdk.response.model', Response::MODEL_ACCOUNT)
     ->label('abuse-limit', 10)
     ->param('userId', '', new CustomId(), 'Unique Id. Choose your own unique ID or pass the string "unique()" to auto generate it. Valid chars are a-z, A-Z, 0-9, period, hyphen, and underscore. Can\'t start with a special char. Max length is 36 chars.')
-    ->param('email', '', new Email(), 'User email.')
-    ->param('password', '', new Password(), 'User password. Must be at least 8 chars.')
+    ->param('email', null, new Email(), 'User email.', true)
+    ->param('password', '', new Password(), 'User password. Must be at least 8 chars.', true)
+    ->param('phone', null, new Phone(), 'Phone number. Format this number with a leading \'+\' and a country code, e.g., +16175551212.', true)
```

Comment:
I don't think we want to allow clients to use this endpoint without an email right ? 

We have other endpoints for this.... Magic URL and anonymous session

Reply:
The idea to not couple this endpoint with an email. For that why would email be required, and phone optional? The other alternative is to keep it the same and add `/email` to the path.

Reply:
But the way it is now, it would allow someone to create an account without passing email, password or phone.. 
If this is the case, how would they be able to create sessions?

Reply:
Yeh, I think this is fine for the users service, for the account service you have a point. What about making it more obvious that this endpoint is coupled with email creation by adding `/email` at the end of the path and changing the name + description a bit?

Reply:
Yes thats a good idea. We just refactored the Create Email Session endpoint to reflect this better
https://github.com/appwrite/appwrite/blob/7723c633a021509b56c30f110917d51058b84cdb/app/controllers/api/account.php#L499